cmake/onnxruntime_unittests.cmake

```diff
@@ -102,7 +102,7 @@ file(GLOB onnxruntime_test_common_src CONFIGURE_DEPENDS
   "${TEST_SRC_DIR}/common/logging/*.h"
   )
 
-if(NOT onnxruntime_MINIMAL_BUILD)
+if(NOT onnxruntime_MINIMAL_BUILD AND NOT onnxruntime_REDUCED_OPS_BUILD)
```

Comment:
If onnxruntime_REDUCED_OPS_BUILD is true, what's the reason to not run the IR tests?

Reply:
IR test enabled.